guides/doc-Project_API/master.adoc

```diff
@@ -0,0 +1,36 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: rest-api
+:rest-api:
+
+= {APIDocTitle}
+
+// This guide is not ready for stable releases
+ifdef::HideDocumentOnStable[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::HideDocumentOnStable[]
```

Comment:
Hidden for foreman-{deb,el}

Reply:
https://github.com/theforeman/foreman-documentation/issues/3249